src/Docker/DatabaseServices.php

```diff
@@ -0,0 +1,64 @@
+<?php
+
+/*
+ * This file is part of the Symfony MakerBundle package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Bundle\MakerBundle\Docker;
+
+/**
+ * @author  Jesse Rushlow <jr@rushlow.dev>
+ *
+ * @internal
+ */
+final class DatabaseServices
+{
+    public static function getDatabaseSkeleton(string $name, string $version): array
+    {
+        switch ($name) {
+            case 'mariadb':
+                return [
+                    'image' => sprintf('mariadb:%s', $version),
+                    'environment' => [
+                        'MYSQL_ROOT_PASSWORD' => 'password',
+                    ],
+                    'ports' => ['3306:3306'],
+                ];
+            case 'mysql':
+                return [
+                    'image' => sprintf('mysql:%s', $version),
+                    'environment' => [
+                        'MYSQL_ROOT_PASSWORD' => 'password',
+                    ],
+                    'ports' => ['3306:3306'],
+                ];
+            case 'postgres':
+                return [
+                    'image' => sprintf('postgres:%s', $version),
+                    'ports' => ['5432:5432'],
```

Comment:
We can (as Fabien does) simplify these to just `5432` - e.g. `ports: [5432]`

Reply:
i'll have to hunt down exactly why, but i ran into a problem on a debian host - if the ports were not mapped out like `[$hostPort:$containerPort]` docker wouldn't expose the port to the host.. I'll reconfirm that edge..

Reply:
If you only specify the container port (5432) docker will choose an ephemeral host port, you run into problems if you choose your database host as localhost.

Reply:
So this is tricky then. The Symfony binary (I believe) automatically detects the random port and uses it to expose the environment variable to the server. In that case, we *do* want the random behavior because it allows you to have multiple database servers (from multiple projects) running on your machine at once. 

This means that this is a decision point that largely depends on whether or not you'll be using the Symfony binary to interface with Docker.

Before we figure out what to do, is my understanding accurate?

Thanks :)